content/asset-tracking/using-the-sdks.textile

```diff
@@ -180,11 +208,32 @@ blang[android].
   )
   ```
 
+  ```[swift]
+  let resolution = Resolution(accuracy: .balanced,
+                              desiredInterval: 1000,
+                              minimumDisplacement: 1.0)
+  let resolutions = DefaultResolutionSet(resolution: resolution)
+  let proximityThreshold = DefaultProximity(spatial: 1)
+  let exampleConstraints = DefaultResolutionConstraints(resolutions: resolutions,
+                                                proximityThreshold: proximity,
+                                                batteryLevelThreshold: 10,
+                                                lowBatteryMultiplier: 2)
+  ```
+
   The next step is create a default "resolution":/asset-tracking#resolution to be used:
 
   ```[android]
   // Prepare the default resolution for the Resolution Policy
-  val defaultResolution = Resolution(Accuracy.BALANCED, desiredInterval = 1000L, minimumDisplacement = 1.0)
+  val defaultResolution = Resolution(Accuracy.BALANCED,
```

Comment:
It should be `accuracy: .balanced` or `accuracy: Accuracy.balanced`

Reply:
@ben-xD This is the Android example, which I believe is correct? See [here](https://github.com/ably/ably-asset-tracking-android#publishing-sdk)

Reply:
Or right I thought it was it was swift